frontend/src/components/Home/HeroSection.jsx

```diff
@@ -7,7 +7,7 @@ import Signup from "../../components/Authentication/Signup";
 
 const HeroSection = () => {
   return (
-    <div className={styles.container}>
+    <div className={`${styles.container}`}>
       <div className={`${styles.gradientBackground}`}>
         <div>
           <div className={styles.content}>
```

Comment:
why the use of backtick \`?
Would it still work without it? If so, then you probably don't want to keep it there
Also inconsistent here `<div className={styles.content}>`